examples/demo_actor/README.md

```diff
@@ -45,12 +66,37 @@ export PYTHONPATH=`pwd`
    ...
    ```
 
+<!-- END_STEP -->
+
+<!-- STEP
+name: Actor Client
+background: true
+sleep: 5
+expected_stdout_lines:
+  - '== APP == call actor method via proxy.invoke_method()'
+  - "== APP == b'null'"
+  - '== APP == call actor method using rpc style'
+  - '== APP == None'
+  - '== APP == call SetMyData actor method to save the state'
```

Comment:
The text presented to the user is much shorter than this one. Also, why sleep only 5 when the command says it will wait for 30 seconds.

Reply:
Ah yes, I neglected to update the output to reflect an actual run. In reality, we're just relying on the 60s timeout, we don't even need the sleep. Will fix.